packages/server/graphql/types/UpdatePromptResponsePayload.ts

```diff
@@ -0,0 +1,33 @@
+import {GraphQLID, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import {makeResolve, resolveNewMeeting} from '../resolvers'
+import makeMutationPayload from './makeMutationPayload'
+import NewMeeting from './NewMeeting'
+import TeamPromptResponse from './TeamPromptResponse'
+
+export const UpdatePromptResponseSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'UpdatePromptResponseSuccess',
+  fields: () => ({
+    promptResponseId: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    meetingId: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    meeting: {
+      type: new GraphQLNonNull(NewMeeting),
+      resolve: resolveNewMeeting
+    },
+    promptResponse: {
+      type: new GraphQLNonNull(TeamPromptResponse),
+      resolve: makeResolve('promptResponseId', 'promptResponse', 'teamPromptResponses')
+    }
+  })
+})
+
+const UpdatePromptResponsePayload = makeMutationPayload(
+  'UpdatePromptResponsePayload',
+  UpdatePromptResponseSuccess
+)
+
+export default UpdatePromptResponsePayload
```

Comment:
-1 we already have this type created using new method [here](https://github.com/ParabolInc/parabol/pull/6333/files#diff-cad85a115f24af217d3957c2fbd0577f226056c7f0b91566ab1e1df42b57ae9bR10)

Reply:
Oops forgot to delete this one! Nice catch!